chromadb/test/ef/test_wordllama_ef.py

```diff
@@ -0,0 +1,7 @@
+from chromadb.utils.embedding_functions.wordllama_embedding_function import WordLlamaEmbeddingFunction
+
+
```

Comment:
can you add:

```py
wordllama = pytest.importorskip("wordllama", reason="wordllama not installed")
```

To make sure the test doesn't fail in CI where the `wordllama` won't be present.